tests/python_functional/functional_tests/source_drivers/file_source/test_acceptance.py

```diff
@@ -20,9 +20,10 @@
 # COPYING for details.
 #
 #############################################################################
-import pytest
 import socket
 
+import pytest
```

Comment:
Empty line between `socket` and `pytest` is necessary?

Reply:
This is because there are 3 types of imports: built-in module, frozen module, own module.
(of course this was an autofix by a linter)

Reply:
Ok I understand now. Thanks!